packages/nns/src/utils/neurons.utils.spec.ts

```diff
@@ -162,25 +177,25 @@ describe("neurons-utils", () => {
         ...proposal,
         ballots: [
           {
-            neuronId: eligibleNeuronsDate[0].neuronId,
+            neuronId: eligibleNeuronsData[0].neuronId,
             vote: Vote.Unspecified,
             votingPower: BigInt(0),
           },
           {
-            neuronId: eligibleNeuronsDate[1].neuronId,
-            vote: Vote.Unspecified,
+            neuronId: eligibleNeuronsData[1].neuronId,
+            vote: Vote.Yes,
             votingPower: BigInt(1),
           },
           {
-            neuronId: eligibleNeuronsDate[2].neuronId,
+            neuronId: eligibleNeuronsData[2].neuronId,
             vote: Vote.Unspecified,
             votingPower: BigInt(0),
           },
         ],
       },
-      neurons: eligibleNeuronsDate,
+      neurons: eligibleNeuronsData,
     });
-    expect(votable.length).toEqual(3);
+    expect(votable.length).toEqual(2);
```

Comment:
OK, here we start with 3 and end up with 2 neurons. So here filtering is being tested.
Maybe the purpose of the above tests isn't to test the filtering?
I'm just pointing it out but I leave it up to you to decide if the coverage is sufficient.

Reply:
These functions do the filtering, so it needs to be covered.